src/App.tsx

```diff
@@ -1,34 +1,57 @@
-/* eslint-disable max-len */
-import React from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { useEffect } from 'react';
+
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { actions as todosActions } from './features/todos';
+
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const todos: Todo[] = useAppSelector(state => state.todos);
+  const todo = useAppSelector(state => state.currentTodo);
+
+  useEffect(() => {
+    getTodos()
+      .then((data) => {
+        dispatch(todosActions.getTodos(data));
+      });
+  }, [dispatch]);
+
   return (
     <>
       <div className="section">
         <div className="container">
           <div className="box">
-            <h1 className="title">Todos:</h1>
+            {todos.length === 0 ? (
```

Comment:
Be careful with such checks, because todos length can be 0 and it doesn't mean you need to show infinite loader. Create separate state `isLoaded` to check if data was loaded, if not - show the loader